docs/puzzles/pooling.md

```diff
@@ -0,0 +1,320 @@
+---
+id: pooling
+title: Pooling
+---
+
+The way that Chia Network does pooling is unlike any blockchain that has come before it. Pool operators actually rely on an on-chain smart contract to verify that they will be able to directly claim any potential pool rewards that farmers create.  This allows pools to trust farmers enough to pay them out, while still keeping the power of making the blocks in the hands of the farmer.  This means that the decentralization of the network remains the same, even as the rewards get concentrated to the pool!
```

Comment:
"...any blockchain..." This seems like a difficult claim to prove... maybe we should undersell a bit.

Reply:
There are over 2500 coins on coinmarketcap.com and I doubt that ANYONE is familiar with all of them.

Reply:
If someone already did this, that would make me sad.  But okay, I'll change it to "unlike many blockchains that have come before it"